pkg/repository/v1/sqlcv1/filters-overwrite.sql

```diff
@@ -0,0 +1,15 @@
+-- name: ListFilters :many
+WITH inputs AS (
+    SELECT
+        UNNEST(@tenantIds::UUID[]) AS tenant_id,
+        UNNEST(@workflowIds::UUID[]) AS workflow_id,
+        -- NOTE: this is nullable, so sqlc doesn't support casting to a type
+        UNNEST(@scopes::TEXT[]) AS scope
+)
+
+SELECT f.*
+FROM v1_filter f
+JOIN inputs i ON (f.tenant_id, f.workflow_id, f.scope) = (i.tenant_id, i.workflow_id, i.scope)
+LIMIT COALESCE(sqlc.narg('filterLimit')::BIGINT, 20000)
```

Comment:
added a limit here just in case but not sure what a reasonable thing for this to be set to is